src/dpp/user.cpp

```diff
@@ -126,6 +126,16 @@ std::string user::get_default_avatar_url() const {
 	}
 }
 
+std::string user::get_avatar_decoration_url(uint16_t size) const {
+	if (this->id) {
+		return utility::cdn_endpoint_url_hash({ i_png },
+											  "avatar-decorations/" + std::to_string(this->id), this->avatar_decoration.to_string(),
```

Comment:
i heard you like indents so i put indents in your indents in your indents in...

Reply:
🤣 wanted to stick with the existing ones